BooruSharp.Others/Pixiv.cs

```diff
@@ -84,7 +84,7 @@ public async Task<byte[]> ImageToByteArrayAsync(SearchResult result)
         public async Task<byte[]> PreviewToByteArrayAsync(SearchResult result)
         {
             var request = new HttpRequestMessage(HttpMethod.Get, result.previewUrl);
-            request.Headers.Add("Referer", result.previewUrl.AbsoluteUri);
+            request.Headers.Add("Referer", result.postUrl.AbsoluteUri);
```

Comment:
Why did you revert previewUrl to postUrl?

Reply:
Well, I didn't revert anything (if you talking about [this](https://github.com/Xwilarg/BooruSharp/commit/a5ea566c4f4a6fbbfe5cefcb58d1db4e73beaf2a#diff-dcc7d9e86bc8bfe040ff2f1ed5866f47R86)), and Pixiv uses original post URL in referer request header, does it not?

Reply:
Oh my bad, I just didn't read things properly